models/DataObject/ClassDefinition/Data/ManyToOneRelation.php

```diff
@@ -734,6 +734,14 @@ public function isFilterable(): bool
         return true;
     }
 
+    public function getInputType() {
```

Comment:
actually could be more detailed defined if only one type has been configured, right?

Reply:
Sure. We could do that.